install/docker-compose.rst

```diff
@@ -88,12 +88,29 @@ Step 4: Start Zammad using DockerHub images
 -------------------------------------------
 
 .. warning::
-   
-   Before starting your containers ensure to not use default login data for 
+
+   Before starting your containers ensure to not use default login data for
    your Zammad database! See Step 3!
 
 .. code-block:: sh
 
    $ docker compose up -d
 
+.. hint:: **🔧 How to run rails/rake commands in containers**
+
+   The docker entrypoint script sets up environment variables required by Zammad to function properly.
+   That is why calling ``rails`` / ``rake`` on the console should be done via one of the following methods:
+
+   .. code-block:: sh
+
+      $ docker compose run --rm zammad-railsserver rails r '...your rails command here...'
+
+   This will run the command via the docker entrypoint and is recommended. In case you require the use of ``docker exec``, you can use the following command:
+
+   .. code-block:: sh
+
+      $ docker exec zammad-docker-compose-zammad-railsserver-1 contrib/docker/docker-entrypoint.sh rails r '...your rails command here...'
```

Comment:
I'd like to suggest to use `/docker-entrypoint.sh` as entrypoint as this is also the entrypoint used by the Dockerfile and significantly shorter :)

Reply:
Good point. Changed accordingly.